shared/i18n/locales/en_US/translation.json

```diff
@@ -200,12 +202,14 @@
   "{{ completed }} of {{ total }} tasks": "{{ completed }} of {{ total }} tasks",
   "Template created, go ahead and customize it": "Template created, go ahead and customize it",
   "Creating a template from <em>{{titleWithDefault}}</em> is a non-destructive action – we'll make a copy of the document and turn it into a template that can be used as a starting point for new documents.": "Creating a template from <em>{{titleWithDefault}}</em> is a non-destructive action – we'll make a copy of the document and turn it into a template that can be used as a starting point for new documents.",
+  "Published": "Published",
+  "Create a published template that's available for use immediately.": "Create a published template that's available for use immediately.",
+  "Save the template within the workspace to enable sharing across collections.": "Save the template within the workspace to enable sharing across collections.",
```

Comment:
I'm kinda uncertain between using "within the workspace" and "at the workspace level".
Or perhaps, a better one? 🤔

Reply:
All templates are technically "_within_ the workspace", so that's probably not the best choice. I think with the UI changes discussed this sentence will go away.